contracts/token/ERC20/extensions/ERC20Crosschain.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.26;
+
+import {ERC20} from "../ERC20.sol";
+import {BridgeERC20Core} from "../../../crosschain/bridges/BridgeERC20Core.sol";
+
+/**
+ * @dev Extension of {ERC20} that makes it natively cross-chain using the ERC-7786 based {BridgeERC20Core}.
+ *
+ * This extension makes the token compatible with:
+ * * {ERC20Crosschain} instances on other chains,
+ * * {ERC20} instances on other chains that are bridged using {BridgeERC20},
+ * * {ERC20Bridgeable} instances on other chains that are bridged using {BridgeERC7802}.
```

Comment:
What does "makes the token compatible with" mean for the points below this sentence?

My understanding is this extension makes the token contract itself also its own bridge.  If that is correct, how does compatibility with other ERC20 tokens fit in?